server/db/postSetup.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS post;
+
+-- CREATE TABLE post (
+--     id serial PRIMARY KEY,
+--     title varchar(255) NOT NULL,
+--     user varchar(100),
+--     body varchar(400) 
+-- );
+
+CREATE TABLE post(
+    id serial PRIMARY KEY,
+    title varchar(255) NOT NULL,
+    username varchar(100),
```

Comment:
You may want to make the username unique - this would also allow for you to potentially further the project with auth & auth.